test/e2e/support/utils.js

```diff
@@ -272,6 +290,17 @@ export async function clickOnMoreMenuItem( buttonLabel ) {
 	await itemButton.click( 'button' );
 }
 
+/**
+ * Clicks on Block Settings Menu item, searches for the button with the text provided and clicks it.
```

Comment:
Aside: Generally speaking we might consider making these utilities less specific to the specific interaction and more toward the task (triggering a block settings menu item). Opens up future refactoring, and we want to avoid expecting a particular mode of interaction (i.e. couldn't this be implemented with a keyboard? And if not, is that a problem we should be aware of?)

Reply:
Also, I don't think we need to rush to add functions to `utils.js`. If we're using them only in one test, it seems fine to define them in that one test, and move things as common patterns emerge.

Reply:
There is already a test using the keyboard shortcut for the menu item, so it might be unnecessary to also have a version that clicks the menu item.

Anyway - I've moved the code out of utils now (for this and the modifier keys mentioned above).